tests/wfm/stitch_test.py

```diff
@@ -95,13 +95,8 @@ def _do_stitching_on_beamline(wavelengths, dim, event_mode=False):
     if event_mode:
         out = wav
     else:
-        out = sc.rebin(wav,
-                       dim='wavelength',
-                       bins=sc.linspace(dim='wavelength',
-                                        start=1.0,
-                                        stop=10.0,
-                                        num=1001,
-                                        unit='angstrom'))
+        out = wav.rebin(wavelength=sc.linspace(
+            dim='wavelength', start=1.0, stop=10.0, num=1001, unit='angstrom'))
```

Comment:
```suggestion
        out = wav.rebin(wavelength=1000)
```
may be good enough for a test?

Reply:
Seems like it isn't. Test fails with your suggestion :-(